static/coffee/controllers.coffee

```diff
@@ -843,6 +845,6 @@ controllers.controller('PodController', ['$scope', 'PodApi', ($scope, PodApi) ->
     # TODO show failure message
 
   $scope.onTriggerSuccess = () ->
-    # TODO update notes
+    $scope.$emit('podActionSuccess')
```

Comment:
Is there a reason we have to emit a `podActionSuccess`, which triggers a `timelineChanged` broadcast? Can we not just have the `timelineChanged` broadcast here?